aiohttp/web_fileresponse.py

```diff
@@ -179,6 +179,7 @@ def prepare(self, request):
         modsince = request.if_modified_since
         if modsince is not None and st.st_mtime <= modsince.timestamp():
             self.set_status(HTTPNotModified.status_code)
+            self._length_check = False
```

Comment:
This line looks hacky but I could live with it.
Looks like `StreamResponse` is too coupled with `Response` class, let's left it for other PR